### PR TITLE
[1st sub-PR of #117] environment files

### PR DESCRIPTION
**Pull request by crystina-z**
sub-PR of #117 

**1. environmental changes  <--**
2. handle other trec format
3. allow sampler to keep the queries with no qrels (for msmarco test set)
4. benchmark validation + better error desc
5. CC set up
6. msmarco passage v1 (core files)

### CODE REVIEW

**Comment by crystina-z** [comment]
force `pip` to be `21.1.3` as the latest pip version seems to hang forever when install pip dependencies

File: requirements.txt [hunk 1]
```diff
@@ -21,6 +21,7 @@ scipy
 sphinx
 sphinxcontrib-apidoc
 recommonmark
+gdown
```
Comment (andrewyates) [inline requirements.txt:24]: I think this change also needs to be made in `setup.py` ?

**Review by andrewyates — CHANGES_REQUESTED** [review]
I think `setup.py` needs to be updated also, but otherwise lgtm!

File: requirements.txt (hunk 1 above)
Comment (crystina-z) [reply requirements.txt:24]: oh yes, updated

**Pull request merged**